test/e2e/liveness.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2015 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"time"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/client"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+	"github.com/golang/glog"
+)
+
+func runLivenessTest(c *client.Client, yamlFileName string) bool {
+	// Read the pod description from the YAML file.
+	podDescr := loadPodOrDie(assetPath("examples", "liveness", yamlFileName))
+	// Randomize the pod name to prevent the test to fail due to problems in clean up from
+	// previous tests or parallel executions of this same test.
+	podName := podDescr.Name + "-" + string(util.NewUUID())
+	podDescr.Name = podName
```

Comment:
I don't know... I kind of like using YAML files from the `examples/` tree since that makes it easier to extend/modify this test and also makes sure our examples are still working... @zmerlynn what do you think?